src/VirtualClient/VirtualClient.Actions/HPLinpack/HPLinpackExecutor.cs

```diff
@@ -0,0 +1,452 @@
+                    $"'{PlatformSpecifics.GetPlatformArchitectureName(PlatformID.Unix, Architecture.X64)}', '{PlatformSpecifics.GetPlatformArchitectureName(PlatformID.Unix, Architecture.Arm64)}'. ",
+                    ErrorReason.PlatformNotSupported);
+            }
+        }
+
+        private async Task ConfigureMakeFileAsync(EventContext telemetryContext, CancellationToken cancellationToken)
+        {
+            string makeFilePath = this.PlatformSpecifics.Combine(this.HPLDirectory, "setup", "Make.Linux_GCC");
+            await this.ExecuteCommandAsync("mv", $"Make.UNKNOWN Make.Linux_GCC", this.PlatformSpecifics.Combine(this.HPLDirectory, $"setup"), telemetryContext, cancellationToken);
+
+            await this.fileSystem.File.ReplaceInFileAsync(
+                    makeFilePath, @"ARCH *= *[^\n]*", "ARCH = Linux_GCC", cancellationToken);
+
+            await this.fileSystem.File.ReplaceInFileAsync(
+                    makeFilePath, @"TOPdir *= *[^\n]*", $"TOPdir = {this.HPLDirectory}", cancellationToken);
+
+            if (this.CpuArchitecture == Architecture.Arm64)
+            {
+                await this.fileSystem.File.ReplaceInFileAsync(
+                    makeFilePath, @"LAdir *=", "LAdir = $(ARMPL_DIR)", cancellationToken);
+
+                await this.fileSystem.File.ReplaceInFileAsync(
+                        makeFilePath, @"LAinc *=", $"LAinc = $(ARMPL_INCLUDES)", cancellationToken);
+
+                await this.fileSystem.File.ReplaceInFileAsync(
+                        makeFilePath, @"LAlib *= *[^\n]*", "LAlib = /opt/arm/armpl_22.1_gcc-11.2/lib/libarmpl.a", cancellationToken);
+
+                await this.fileSystem.File.ReplaceInFileAsync(
+                        makeFilePath, @"LINKER *= *[^\n]*", "LINKER = mpifort", cancellationToken);
+
+                await this.fileSystem.File.ReplaceInFileAsync(
+                        makeFilePath, @"CCFLAGS *= *[^\n]*", "CCFLAGS = $(HPL_DEFS) -Ofast -march=armv8-a", cancellationToken);
+            }
+            else if (this.CpuArchitecture == Architecture.X64)
+            {
+                await this.fileSystem.File.ReplaceInFileAsync(
+                        makeFilePath, @"MPinc *=", $"MPinc =  -I/usr/lib/x86_64-linux-gnu/openmpi", cancellationToken);
+
+                await this.fileSystem.File.ReplaceInFileAsync(
+                        makeFilePath, @"MPlib *=", "MPlib =  /usr/lib/x86_64-linux-gnu/openmpi/lib/libmpi.so", cancellationToken);
+
+                await this.fileSystem.File.ReplaceInFileAsync(
+                        makeFilePath, @"LAinc *=", "LAinc = -I/usr/lib/x86_64-linux-gnu", cancellationToken);
+            }
+
+            await this.fileSystem.File.ReplaceInFileAsync(
+                    makeFilePath, @"CC *= *[^\n]*", "CC = mpicc", cancellationToken); 
+        }
+
+        private async Task ConfigureHPLdatFileAsync(EventContext telemetryContext, CancellationToken cancellationToken)
```

Comment:
Naming: ConfigureDatFileAsync

Unless there is another .dat file involved in the operations workflow, there is not a real need to call out the HPL part given the developer is working in an executor that makes that part relatively clear.